### DISCUSSION TITLE
Report

### DISCUSSION THREAD

**Discussion by holguer209** [celo-org/staked-celo-web-app#268] [Polls]
![Screenshot_20241017-171552](https://github.com/user-attachments/assets/894f0f75-a06f-408d-a9d8-37d037cb068c)
![Screenshot_20241017-191116](https://github.com/user-attachments/assets/e8ac8c39-283b-4de8-a7ef-afe957b6f4f7)
I did not really that you people did not want to send me my order

**Comment by Mulyana529** [comment]
> ![Screenshot_20241017-171552](https://github.com/user-attachments/assets/894f0f75-a06f-408d-a9d8-37d037cb068c)
> ![Screenshot_20241017-191116](https://github.com/user-attachments/assets/e8ac8c39-283b-4de8-a7ef-afe957b6f4f7)
> I did not really that you people did not want to send me my order
